test/iidm/CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ set(UNIT_TEST_SOURCES
 )
 
 add_executable(unit-tests-iidm ${UNIT_TEST_SOURCES})
-target_compile_definitions(unit-tests-iidm PRIVATE BOOST_TEST_DYN_LINK)
-target_link_libraries(unit-tests-iidm PRIVATE iidm-tests Boost::unit_test_framework)
+target_link_libraries(unit-tests-iidm PRIVATE iidm-tests Boost::unit_test_framework iidm)
```

Comment:
It should not be necessary to pass iidm and unit_test_framework because they are linked to iidm-tests.